app/views/templates/search/search_object.rb

```diff
@@ -175,100 +189,99 @@ def content
               }
             },
             origin: {
-              url:                 render_document_show_field_value(document, 'aggregations.edmIsShownAt'),
-              institution_name:    render_document_show_field_value(document, 'aggregations.edmDataProvider'),
+              url: render_document_show_field_value(document, 'aggregations.edmIsShownAt'),
+              institution_name: render_document_show_field_value(document, 'aggregations.edmDataProvider'),
               institution_country: render_document_show_field_value(document, 'europeanaAggregation.edmCountry'),
-              content_present:     collect_values(['aggregations.edmDataProvider', 'europeanaAggregation.edmCountry']).length > 0
             },
             people: data_section({
-              :title => 'site.object.meta-label.people',
-              :sections  => [
+              title: 'site.object.meta-label.people',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.